var/spack/repos/builtin/packages/openfoam/package.py

```diff
@@ -369,8 +369,9 @@ class Openfoam(Package):
     # See https://github.com/spack/spack/pull/22303 for reference
     depends_on(Boost.with_default_variants)
 
-    # OpenFOAM does not play nice with CGAL 5.X
-    depends_on("cgal@:4")
+    # Earlier versions of OpenFOAM did not play nice with CGAL 5.X
+    depends_on("cgal")
```

Comment:
Does the constraint still apply to other versions listed in the package?  If so, then a conditional dependency is in order (see https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-dependencies).

Reply:
I made the dependency conditional on the version.